django_lightweight_queue/management/commands/queue_resume.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+
+from django.core.management.base import BaseCommand, CommandError
+
+from ...utils import get_backend
+
+QueueName = str
+
+
+class Command(BaseCommand):
+    help = """
+    Command to resume work immediately on a redis-backed queue.
+
+    This removes a pause which may be in place for the given queue, though it
+    may not cause workers to resume work immediately.
+    """  # noqa:A003 # inherited name
+
+    def add_arguments(self, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument(
+            'queue',
+            action='store',
+            help="The queue to resume.",
+        )
+
+    def handle(self, queue: QueueName, **options: object) -> None:
+        backend = get_backend(queue)
+
+        if not hasattr(backend, 'resume'):
+            raise CommandError(
+                "Configured backend '{}.{}' doesn't support pausing".format(
```

Comment:
Should this be _resuming_?

This highlight that the duck typing here allows a backend to support pausing without resuming (kinda makes sense) or resuming without pausing (absurd) - I wonder if we could encode this in the classes in a way that you need both or neither? Maybe a `PausableBackend` ABC which the redis backend inherit from?

Reply:
b8ad26c fixes the error message.
Will have a look at using an ABC or something here, though I suspect there'll be conflicts with #42 so it might be easier to get that merged first.

Reply:
602c7b4 introduces custom backends. I've left it possible to implement a backend which only support pausing, which I think still makes sense given the need to provide a deadline, though I suspect most would want to implement a pause-resume backend.